x-pack/legacy/plugins/code/public/sagas/editor.ts

```diff
@@ -199,13 +198,6 @@ function* handleMainRouteChange(action: Action<Match>) {
   }
   const tree = yield select(getTree);
   const isDir = pathType === PathTypes.tree;
-  const openPath = isDir
-    ? file
-    : (file || '')
-        .split('/')
-        .slice(0, -1)
-        .join('/');
-  yield put(openTreePath(openPath || ''));
```

Comment:
Can't remember why this was added at first place, could you please double check deleting this line won't cause other problem?